layers/+lang/go/packages.el

```diff
@@ -25,7 +25,8 @@
   '(
     company
     dap-mode
-    (company-go :requires company)
+    (company-go :requires company
```

Comment:
I think you need to also cover the case when there is no go backend.

Reply:
Sorry my mistake some layers don't set this var reliably and have some code for inferring if lsp is selected from the layer configuration if this is not the case here its ok.